src/translators/sql_translator.py

```diff
@@ -1411,11 +1427,7 @@ def _format_response(
                     elif original_name == self.TIME_INDEX_NAME:
                         v = self._get_isoformat(v)
                         if single_value:
-                            n = {
-                                'value': v,
-                                'type': 'DateTime'
-                            }
-                            e.setdefault('dateModified', n)
+                            e.setdefault('dateModified', v)
```

Comment:
i think you are using as base an old version of the code

Reply:
> i think you are using as base an old version of the code

@chicco785  , I am working on this and I will update this PR.